versionist.conf.js

```diff
@@ -1,32 +1,59 @@
-module.exports = {
+var _ = require('lodash');
+var execSync = require('child_process').execSync;
 
-  subjectParser: 'angular',
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
 
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
+  editChangelog: true,
+  parseFooterTags: true,
   getGitReferenceFromVersion: 'v-prefix',
+  incrementVersion: 'semver',
+  updateVersion: 'npm',
 
+  // Always add the entry to the top of the Changelog, below the header.
   addEntryToChangelog: {
     preset: 'prepend',
-    fromLine: 5
+    fromLine: 6
   },
 
-  updateVersion: 'npm',
-
+  // Only include a commit when there is a footer of 'change-type'.
```

Comment:
"... a footer of 'change-type' or a footer of 'changelog-entry'." ?
(unless I've misunderstood again)

And should it be "a footer **tag** of 'change-type'" ?